tests/RequestBuilder/SortingRequestBuilderTest.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\RequestBuilder;
+
+use Fig\Http\Message\RequestMethodInterface;
+use Lmc\Matej\Exception\LogicException;
+use Lmc\Matej\Http\RequestManager;
+use Lmc\Matej\Model\Command\Interaction;
+use Lmc\Matej\Model\Command\Sorting;
+use Lmc\Matej\Model\Command\UserMerge;
+use Lmc\Matej\Model\Request;
+use Lmc\Matej\Model\Response;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Lmc\Matej\RequestBuilder\SortingRequestBuilder
+ * @covers \Lmc\Matej\RequestBuilder\AbstractRequestBuilder
+ */
+class SortingRequestBuilderTest extends TestCase
+{
+    /** @test */
+    public function shouldBuildRequestWithCommands(): void
+    {
+        $builder = new SortingRequestBuilder('userId1', ['itemId1', 'itemId2']);
+
+        $interactionCommand = Interaction::detailView('userId1', 'itemId1');
+        $builder->addInteraction($interactionCommand);
+
+        $userMergeCommand = UserMerge::mergeFromSourceToTargetUser('sourceId1', 'targetId1');
+        $builder->addUserMerge($userMergeCommand);
+
+        $request = $builder->build();
+
+        $this->assertInstanceOf(Request::class, $request);
+        $this->assertSame(RequestMethodInterface::METHOD_POST, $request->getMethod());
+        $this->assertSame('/sorting', $request->getPath());
+
+        $requestData = $request->getData();
+        $this->assertCount(3, $requestData);
+        $this->assertContains($interactionCommand, $requestData);
```

Comment:
We should check the exact command order here.

Ie. index 0 is interaction, index 1 user mege, index 2 is sorting command.

```php
$this->assertSame($interactionCommand, $requestData[0]); 
$this->assertSame($userMergeCommand, $requestData[1]); 
$this->assertSame($sortingCommand, $requestData[2]); 
```

Also the sorting command check below would then not be necessary, wouldn't it?